docs/building-your-quest/actions/run-command.md

```diff
@@ -0,0 +1,63 @@
+---
+title: run_command
+parent: Actions
+grand_parent: Building Your Quest
+---
+
+# run_command
+
+Category: Remote Machine
+Description: Run a command on the user's remote codespace
+Type: Action
+
+### Description
+
+**Type**: action
+
+Run a command on the user's codespace. The action uses an agent running on the remote machine.
+
+## Params
+
+- **command:** The command to run on the remote machine. i.e. `ls -l`
+- **timeout**: The maximal time in seconds to wait for the command to finish. The default is 60 seconds.
```

Comment:
30 seconds, I think that makes more sense

Reply:
> 30 seconds, I think that makes more sense

changed

Reply:
approved